pkg/rewards/rewards.go

```diff
@@ -413,6 +414,21 @@ func (rc *RewardsCalculator) generateGoldTables(snapshotDate string) error {
 		return err
 	}
 
+	if err := rc.Generate7ActiveODRewards(snapshotDate); err != nil {
+		rc.logger.Sugar().Errorw("Failed to generate active od rewards", "error", err)
+		return err
+	}
+
+	if err := rc.GenerateGold8OperatorODRewardAmountsTable(snapshotDate); err != nil {
+		rc.logger.Sugar().Errorw("Failed to generate operator od reward amounts", "error", err)
+		return err
+	}
+
```

Comment:
Is table 9 (Table_9_StakerODRewardAmounts) still under development?

Reply:
Added in [49d6bc7](https://github.com/Layr-Labs/sidecar/pull/131/commits/49d6bc71da1b62b0cb5acd3c434758e7c42bd4be)